package.json

```diff
@@ -38,14 +38,15 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "@commercetools/sync-actions": "^2.0.0",
+    "@commercetools/sync-actions": "^2.2.1",
```

Comment:
Where do you use it?

Reply:
This is being used when syncing prices - https://github.com/sphereio/sphere-product-import/blob/master/lib/price-import.coffee#L19